Makefile

```diff
@@ -36,6 +36,8 @@ CONTAINER_REGISTRY ?= quay.io/jetstack
 
 GOPROXY ?= https://proxy.golang.org,direct
 
+KUBECONFIG ?= $(BINDIR)/kubeconfig.yaml
```

Comment:
What is the purpose of this variable? As a user I would expect that if I pass a custom path here, the current cluster from that kubeconfig will be used. However, that is not the case, this will always result in a kind cluster named 'trust' will be created, my local kubeconfig will be modified as well as one will be created in $BINDIR.
Should we just keep the kubeconfig hardcoded to not make folks think they can use this var to parameterize it?

Reply:
> What is the purpose of this variable? 

Originally my idea was to allow users to put in their own kubeconfig, but you make a valid point - that won't work without more work.

>  Should we just keep the kubeconfig hardcoded to not make folks think they can use this var to parameterize it?

I agree with this yeah, makes sense. Great review comment, thanks!